masterlist.yaml

```diff
@@ -314,6 +314,10 @@ common:
     name: 'skse/plugins/po3_SpellPerkItemDistributor.dll'
     display: '[Spell Perk Item Distributor](https://www.nexusmods.com/skyrimspecialedition/mods/36869/)'
 
+  - &DAR
```

Comment:
Please sort DAR alphabetically inbetween the other File Anchors, so after `&BaseObjSwap` and before `&ENBSeries`.